src/PlusDataCollection/Capistrano/vtkPlusCapistranoVideoSource.h

```diff
@@ -26,22 +26,22 @@ class vtkPlusDataCollectionExport vtkPlusCapistranoVideoSource: public vtkPlusUs
   /*! Constructor for a smart pointer of this class*/
   static vtkPlusCapistranoVideoSource* New();
   vtkTypeMacro(vtkPlusCapistranoVideoSource, vtkPlusDevice);

```

Comment:
Would it be possible to change the base type to be vtkPlusUsDevice

Reply:
I just did that, and it resolves the issue.